### ISSUE TITLE
"Invalid line in environment file" on empty line

### ISSUE THREAD

**Issue by ava-dylang** [docker/docker-py#1411]
```
docker==2.0.2
docker-compose==1.10.0
docker-py==1.10.6
docker-pycreds==0.2.1
dockerpty==0.4.1
dockerrotate==3.0
Python 2.7.12
Client:
 Version:      1.12.1
 API version:  1.24
 Go version:   go1.6.3
 Git commit:   23cf638
 Built:        Thu Aug 18 05:33:38 2016
 OS/Arch:      linux/amd64

Server:
 Version:      1.12.1
 API version:  1.24
 Go version:   go1.6.3
 Git commit:   23cf638
 Built:        Thu Aug 18 05:33:38 2016
 OS/Arch:      linux/amd64
```

the problem happens when there is a blank newline in the vars file (that isn't the last newline in the file)

**Comment by shin-** [comment]
Thank you for the report!

**Comment by shin-** [comment]
I believe this was fixed by #1317
